assets/scss/main.scss

```
@@ -27,8 +27,8 @@
     h5[id]:before {
         display: block;
         content: " ";
-        margin-top: 0; // NK - override for default values, to allow selection of items right above headers
-        height: 0; // NK - override for default values, to allow selection of items right above headers
+        margin-top: -5; // NK - set back to default to prevent MxDock from going on top of headings
+        height: 5; // NK - set back to default to prevent MxDock from going on top of headings
```

Did we decide to make this 48px to match the height of MxDock?
```suggestion
        margin-top: -48px; // NK - set back to default to prevent MxDock from going on top of headings
        height: 48px; // NK - set back to default to prevent MxDock from going on top of headings
```

---

Yes, we did agree on this, slipped my mind. I will adjust and test.